libs/doc-components/src/components/docTokenTable/docTokenTable.tsx

```diff
@@ -29,34 +29,51 @@ const applyStyle = (category: string, value: string, use?: string): React.CSSPro
   const style: React.CSSProperties = {};
   const categoryStyles = categoryStyleMapping[category];
 
+  if (value.endsWith('-@')) console.log(value)
```

Comment:
Was this meant to be left here?

Reply:
Nope, I removed it and then realized I never saved and committed the file lol.